test/features.jl

```diff
@@ -164,7 +164,7 @@ D(f, x) = grad(f, x)[1]
 @test D(x -> x*D(y -> x*y, 1), 4) == 8
 
 if VERSION >= v"1.1"
-  @test sin'''(1.0) ==  -cos(1.0)
+  @test Zygote.gradient(y -> Zygote.gradient(x -> Zygote.gradient(sin, x)[1], y)[1], 1.0)[1] ==  -cos(1.0)
```

Comment:
Would be better to use the `D` operator here.

Reply:
@MikeInnes What is the syntax of the `D` operator?

Reply:
It's defined and used right above; it's essentially just a one-argument version of `gradient`.